MIGRATION-GUIDE-TOOLKIT.md

```diff
@@ -0,0 +1,107 @@
+# Migration from @axa-fr/react-toolkit
+
+If you are using `@axa-fr/react-toolkit` and want to migrate to
+`@axa-fr/design-system`, you can follow this guide. Most components work the
+same way, and migrating _should_ be painless. There are however a few
+exceptions.
+
+## React 17
+
+Support for react 17 has been dropped. This package now requires react 18 or
+higher.
+
+## 👋 CSS imports
+
+Importing CSS is no longer required as components bring their own styles. You
+can remove the following line from your project:
+
+```diff
+- import '@axa-fr/react-toolkit-all/dist/style/af-toolkit-core.css';
+- import '@axa-fr/react-toolkit-core/dist/assets/fonts/icons/af-icons.css';
+```
+
+However we now require sass support in your project. If you don't have it, you
+can install it with:
+
+```bash
+npm install sass
+```
+
+If you are using vite, it will automatically pickup on sass being present.
+
+## Alert
+
+The default icon for the `Alert` in error component has changed. It now uses the
+glyphicon `glyphicon-minus-sign` instead of `glyphicon-exclamation-sign`.
+
+Before:
+
+![Alert uses glyphicon-exclamation-sign](./docs/images/migration-slash-alert-before.webp)
+
+After:
+
+![Alert uses glyphicon-minus-sign](./docs/images/migration-slash-alert-after.webp)
+
+## 🚮 CardGroupCheckbox and CardGroupRadio
```

Comment:
peut rassembler dans un section "component will be removed" et lister ce serait plus simple non ?